### ISSUE TITLE
Translations to other languages?

### ISSUE THREAD

**Issue by henrikfalk** [liamcain/obsidian-calendar-plugin#164]
It would be nice with support for translations to other languages.

Settings, dialogs etc.

I can translate to Danish. I have done that with Obsidian and the Obsidian docs.

**Comment by liamcain** [comment]
I would love to see settings and dialogs get translated! I'll check with the Obsidian devs to see there's anything they can expose via the API to help with plugin localization.

**Comment by henrikfalk** [comment]
Yes, I was actually wondering if "plugin localization" maybe was implemented or a feature request is needed.

**Comment by henrikfalk** [comment]
Hi again :)
There is several plugins now implementing localization. It seems most of them is based on the kanban plugin: https://github.com/mgmeyers/obsidian-kanban
Maybe you could use that code to localize your plugin?
I use the same code in the plugins that I develop, and it works nicely :)
